.circleci/config.yml

```diff
@@ -0,0 +1,104 @@
+---
+version: 2.1
+
+orbs:
+  prometheus: prometheus/prometheus@0.1.0
```

Comment:
it could be  `prometheus/prometheus@0.4.0`

Reply:
Ahh, I just copy-pasta'd from node_exporter. I guess we need to sync these.

We should bump the major version to `1.0.0`, then we can use this syntax:
```yaml
orbs:
  prometheus: prometheus/prometheus@1
```